src/main/kotlin/no/nav/syfo/cronjob/VerifyBehandlereForKontorCronjob.kt

```diff
@@ -286,15 +286,15 @@ class VerifyBehandlereForKontorCronjob(
         existingBehandlereForKontor: List<PBehandler>,
     ) {
         aktiveBehandlereForKontor.filter {
-            it.hprId != null && BehandlerKategori.fromKategoriKode(it.kategori) != null
+            it.hprId != null
```

Comment:
Viser seg at kategori mangler for mange behandlere i Adresseregisteret, men vi bør likevel oppdatere navn og herId.